### PR TITLE
feat: Annotate home screen queries with optionalField directive

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Feature**

This PR resolves [CX-1595]

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

[CX-1595]: https://artsyproduct.atlassian.net/browse/CX-1595

### CODE REVIEW

File: src/lib/Scenes/Home/Home.tsx [hunk 1]
```diff
@@ -38,11 +38,11 @@ interface Props extends ViewProps {
...
-  const artworkModules = homePage.artworkModules || []
-  const salesModule = homePage.salesModule
-  const collectionsModule = homePage.marketingCollectionsModule
-  const artistModules = (homePage.artistModules && homePage.artistModules.concat()) || []
-  const fairsModule = homePage.fairsModule
+  const artworkModules = homePage?.artworkModules || []
+  const salesModule = homePage?.salesModule
+  const collectionsModule = homePage?.marketingCollectionsModule
+  const artistModules = (homePage?.artistModules && homePage.artistModules.concat()) || []
+  const fairsModule = homePage?.fairsModule
```
Comment (MounirDhahri) [inline src/lib/Scenes/Home/Home.tsx:45]: **praise:** clever 💫

**Review by MounirDhahri — APPROVED** [review]

**Review by pvinis — APPROVED** [review]
nice

**Review by gkartalis — APPROVED** [review]
Looking good! 👌

**Comment by MounirDhahri** [comment]
🛳🛳🛳

**Pull request merged**